PHP/PHPhomework3.php

```diff
@@ -0,0 +1,18 @@
+<?php
+   
+function primeNumberFinder($n)
+{
+    $x = 0;
+    $y = 1;
+    for ($i = 2; $i < $n; $i++) {
+        $x = $n % $i;
+        $y *= $x;
+    }
+    if($y == 0) {
+        echo "The number is not prime number.";
+    } else {
+        echo "The number is prime number.";
+    }
+}
+primeNumberFinder();
```

Comment:
@toru828
I think you missed `$n`

Reply:
Where should I put $n?

Reply:
`primeNumberFinder($n) {...}`
you should call with $n such as `primeNumberFinder(101);`